Bor/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿// <copyright file="Program.cs" company="Alina">
+// Copyright (c) Alina. All rights reserved.
+// </copyright>
+
+using Bor;
+
+if (! new Tests().RunTests())
```

Comment:
```suggestion
if (!new Tests().RunTests())
```
Унарные операторы пробелом не отделяются